src/main/java/com/rcv/Main.java

```diff
@@ -31,6 +31,7 @@
   // param: args command line argument array
   // returns: N/A
   public static void main(String[] args) {
+    System.out.println("RCVRC Universal Tabulator v 1.0");
```

Comment:
let's put the app name and version in constants

Reply:
Agree, and please set version to 1.0.0.

Reply:
fixed